app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/NewTransferViewModel.kt

```diff
@@ -179,6 +183,15 @@ class NewTransferViewModel @Inject constructor(
     }
     //endregion
 
+    //region Password
+    var transferPassword by mutableStateOf("")
+
+    val isPasswordValid by derivedStateOf {
+        val trimmedPassword = transferPassword.trim()
```

Comment:
Should we not allow spaces on the password field ? I'm afraid this trim might cause issues.

Reply:
That sound like a bad practice from their side no ?

Reply:
We'll talk about entirely blocking users from adding space on team meeting

Reply:
DO we want to keep the password as is ? The user want a specific password and modifying it is dangerous ?